test/cctest/test_inspector_socket_server.cc

```diff
@@ -575,3 +583,45 @@ TEST_F(InspectorSocketServerTest, TerminatingSessionReportsDone) {
   socket1.ExpectEOF();
   SPIN_WHILE(!delegate.done);
 }
+
+TEST_F(InspectorSocketServerTest, FailsToBindToNodejsHost) {
+  TestInspectorServerDelegate delegate;
+  ServerHolder server(&delegate, &loop, "nodejs.org", 0, nullptr);
+  ASSERT_FALSE(server->Start());
+  SPIN_WHILE(uv_loop_alive(&loop));
+}
+
+bool has_ipv6_address() {
+  uv_interface_address_s* addresses;
+  int address_count;
+  int err = uv_interface_addresses(&addresses, &address_count);
```

Comment:
done. I'm bit fuzzy on this - but should uv_free_* be called if respective call (in this case to `uv_interface_addresses`) failed?